compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -475,6 +476,10 @@ void CircleOptimizer::optimize(loco::Graph *g) const
   {
     phase.emplace_back(std::make_unique<luci::RemoveUnnecessaryAddPass>());
   }
+  if (_options->query(Options::Algorithm::RemoveUnnecessaryCast))
+  {
+    phase.emplace_back(std::make_unique<luci::RemoveUnnecessaryCastPass>());
+  }
```

Comment:
Note: update with regards to https://github.com/Samsung/ONE/pull/13800 when it's merged